CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.1.0
+
+* Add generic arguments to `run` in `LoadBalancer` and `IsolateRunner`.
```

Comment:
I don't see this change?

Reply:
In previous PR – including here for completeness